pkg/asset/manifests/openshift.go

```diff
@@ -177,11 +177,6 @@ func (o *Openshift) Generate(dependencies asset.Parents) error {
 		assetData["99_role-cloud-creds-secret-reader.yaml"] = applyTemplateData(roleCloudCredsSecretReader.Files()[0].Data, templateData)
 	}
 
-	switch platform {
-	case openstacktypes.Name:
-		assetData["99_rolebinding-cloud-creds-secret-reader.yaml"] = applyTemplateData(roleBindingCloudCredsSecretReader.Files()[0].Data, templateData)
```

Comment:
it looks like this was the only consumer of asset `roleBindingCloudCredsSecretReader` can we remove `roleBindingCloudCredsSecretReader` too?

Reply:
sure, will be done!